src/App.tsx

```diff
@@ -1,14 +1,39 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [preparedTodos, setPreparedTodos] = useState<Todo[]>([]);
+  const [sorted, setSorted] = useState('');
+  const [filter, setFilter] = useState('');
+  const [modalTodo, setModalTodo] = useState<Todo | undefined>();
+
+  useEffect(() => {
+    getTodos().then(data => {
+      setTodos(data);
+      setPreparedTodos(data);
+    });
+  }, []);
+
+  useEffect(() => {
+    if (sorted === 'active') {
+      setPreparedTodos(todos.filter(todo => !todo.completed && todo.title.includes(filter.toLocaleLowerCase())));
```

Comment:
I think you don't need to use `toLocaleLowerCase() `here just use standard `toLowerCase()`